package.json

```diff
@@ -47,17 +47,26 @@
         "test": "jest",
         "test:watch": "jest --watch",
         "debug": "node --inspect-brk node_modules/.bin/jest -i",
-        "clean": "git clean -f -d test"
+        "clean": "git clean -f -d test",
+        "prepare": "conditional-install"
     },
     "engines": {
         "node": ">=10.0"
     },
     "dependencies": {
-        "ilib": "^14.18.0",
+        "ilib": "^14.19.0",
         "json5": "^2.2.3"
     },
     "devDependencies": {
-        "jest": "^26.6.3",
+        "conditional-install": "^1.0.1",
         "loctool": "^2.23.1"
+    },
+    "conditionalDependencies": {
+        "process.versions.node < 14.0.0": {
+            "jest": "^26.0.0"
+        },
+        "process.versions.node >= 14.0.0": {
+            "jest": "^29.0.0"
```

Comment:
This is more of a comment for the conditional-install package, but maybe there could be a default jest 29 devDependency and a conditional would later override it with 26? I feel like this would better indicate that jest is in fact a required dependency. Not sure if that package supports such behaviour though.